src/amp-ads/40_Experimental_Ads/Scrollbound_Animation_Ad.html

```diff
@@ -0,0 +1,191 @@
+        width: 340px;
+        height: 240px;
+        position: relative;
+        margin: 0 auto;
+        display: block;
+      }
+      /* Position the footer absolutely, at the bottom of the container. */
+      .footer {
+        background: #000;
+        font-size: 12px;
+        padding: 8px;
+        text-transform: uppercase;
+        display: flex;
+        align-items: center;
+        color: #fff;
+        z-index: 2;
+        position: absolute;
+        left: 0;
+        bottom: 0;
+        right: 0;
+      }
+      .footer-logo {
+        border-right: 1px solid #fff;
+        padding: 0 8px 0 0;
+        margin: 0 8px 0 0;
+      }
+      .logo-img {
+        display: block;
+      }
+      .stretch {
+        flex: 1;
+      }
+      .button {
+        text-transform: uppercase;
+        padding: 8px;
+        color: #fff;
+        display: inline-block;
+        background-color: #2979ff;
+        text-decoration: none;
+      }
+      .button:active {
+        background-color: #92bbff;
+      }
+      /* Basic cards styling */
+      .cards-container {
+        position: absolute;
+        z-index: 1;
+        left: 0;
+        right: 0;
+      }
+      .card {
+        width: 100%;
+        overflow: hidden;
+        position: relative;
+      }
+      /* Change position of the expand button for the middle card */
+      .card-gauges .expand-button {
+        left: 90px;
+        top: 50px;
+      }
+      .learn-more a:active {
+        background-color: #f0f0f0;
+      }
+    </style>
+  </head>
+  <body>
+    <!-- ## Animations -->
+    <!--
+    The animations are implemented using [amp-animation](https://www.ampproject.org/docs/reference/components/amp-animation) which is an AMP component that uses [Web Animations API](https://www.w3.org/TR/web-animations/) to support both __time__ and __scroll__ based animations.
+    -->
+    <amp-animation id="adAnim" layout="nodisplay">
+      <script type="application/json">
+        {
+          "duration": "1s",
+          "fill": "both",
+          "direction": "alternate",
+          "animations": [
+            {
+              "selector": "#cardsContainer",
+              "keyframes": [
+                { "transform": "translateY(0)" },
+                { "transform": "translateY(-384px)"}
+              ]
+            }
+          ]
+        }
+      </script>
+    </amp-animation>
+    <!--
+    **Slide cards opposite to scroll direction**:
+    The ad initially shows the top-most card. As the document scrolls, the other two cards are revealed.
+
+    To achieve that, we define 4 keyframes. Between 0% and 10% of viewport, the first card is displayed. Between 10% and 90%, the cards container will vertically translate to -384 pixels, which is the negative of double as each card's height. When reaching this point, the third card will be fully displayed. Between 90% and 100%, the same position is kept.
+    -->
+
+    <!-- ## Scrollbound Animations -->
```

Comment:
@aghassemi PTAL at how I use `<amp-animation>` and `<amp-position-observer>` here, and the documentations.

Reply:
LGTM. thanks for fixing these!